os/Makefile.unix

```diff
@@ -469,6 +469,16 @@ endif
 endif
 endif
 
+ifneq ($(findstring "smartfs",$(CONFIG_FLASH_PART_TYPE)), "smartfs")
```

Comment:
This is not to make smartfs image for user binary. It's just making empty smartfs image.

Reply:
@jeongchanKim Could you check belows?
1. Let's call these with some config. I mean when user enables new config, then let's make empty smartfs image.
    User could make it with some content so that it should be disabled by user.
2. Because this is not for user binary, there is no reason to be in the pass1 target. Let's move into the post target.